packages/react-devtools-shared/src/backend/legacy/renderer.js

```diff
@@ -649,6 +649,25 @@ export function attach(
     }
   }
 
+  function storeAsGlobal(id: number, path: Array<string | number>): void {
+    const inspectedElement = inspectElementRaw(id);
+    if (inspectedElement !== null) {
+      const value = getInObject(inspectedElement, path);
+
+      window.$reactTemp = value;
```

Comment:
Maybe autoincrement like the browser does? So it's easy to save multiple.

Reply:
I considered that but then I wasn't sure there was a compelling use case for saving multiple, and it gets harder to coordinate with multiple roots/renderers. Seems easy enough to re-assign (in console) if you want to save a second and compare.

Reply:
I guess we could pass the auto-incremented value from the frontend...

Reply:
I guess the memory concern isn't all that big, since this is a debugging use case. I've added the auto-incremented behavior.

Reply:
After some thought, I think storing separate variable can be practically useful, since you could store a value at one point, then store again after an update, then compare the two values in the console to see what changed.